examples/scenes/src/mmark.rs

```diff
@@ -118,13 +119,13 @@ impl TestScene for MMark {
 }
 
 const COLORS: &[Color] = &[
-    Color::from_rgba8(0x10, 0x10, 0x10, 0xff),
-    Color::from_rgba8(0x80, 0x80, 0x80, 0xff),
-    Color::from_rgba8(0xc0, 0xc0, 0xc0, 0xff),
-    Color::from_rgba8(0x10, 0x10, 0x10, 0xff),
-    Color::from_rgba8(0x80, 0x80, 0x80, 0xff),
-    Color::from_rgba8(0xc0, 0xc0, 0xc0, 0xff),
-    Color::from_rgba8(0xe0, 0x10, 0x40, 0xff),
+    palette::css::BLACK,
+    palette::css::GRAY,
+    palette::css::LIGHT_GRAY,
+    palette::css::BLACK,
+    palette::css::GRAY,
+    palette::css::LIGHT_GRAY,
+    palette::css::DEEP_PINK,
```

Comment:
These need to match the [motionmark benchmark](https://github.com/WebKit/MotionMark/blob/baf37c1fcb690abbe048e249e248e7847bddd34f/MotionMark/tests/core/resources/canvas-tests.js#L75-L76)